packages/techdocs-common/src/stages/publish/awsS3.test.ts

```diff
@@ -41,13 +44,15 @@ const createMockEntityName = (): EntityName => ({
   namespace: 'test-namespace',
 });
 
+const rootDir = os.platform() === 'win32' ? 'C:\\rootDir' : '/rootDir';
+
 const getEntityRootDir = (entity: Entity) => {
   const {
     kind,
     metadata: { namespace, name },
   } = entity;
-  const entityRootDir = path.join(namespace as string, kind, name);
-  return entityRootDir;
+
+  return path.join(rootDir, namespace as string, kind, name);
```

Comment:
it would be more safe to do `namespace || ENTITY_DEFAULT_NAMESPACE` (from the catalog model)